app/models/container_image.rb

```diff
@@ -95,6 +95,8 @@ def annotate_deny_execution(causing_policy)
       "security.manageiq.org/failed-policy" => causing_policy,
       "images.openshift.io/deny-execution"  => "true"
     )
+  rescue ManageIQ::Providers::Openshift::ContainerManagerMixin::KubeException => e
+    _log.error("#{__method__} #{e.message}")
```

Comment:
@enoodle I have the feeling that if we're suppressing an exception we should be more specific. E.g. in the case we know we can't handle we know the error would be a 404?

Reply:
I have a dumber question: what's `ManageIQ::Providers::Openshift::ContainerManagerMixin::KubeException`?
I'm getting "NameError: uninitialized constant" on master.